src/main/java/tc/oc/pgm/rotation/FixedPGMMapOrder.java

```diff
@@ -0,0 +1,160 @@
+public class FixedPGMMapOrder implements PGMMapOrder {
+  private ConfigurationSection configurationSection;
+
+  private String name;
+  private boolean enabled;
+  private List<PGMMap> maps = new ArrayList<>();
+  private int players;
+  private int position;
+  private boolean booting = true;
+
+  FixedPGMMapOrder(ConfigurationSection configurationSection, String name) {
+    this.configurationSection = configurationSection;
+    this.name = name;
+    this.enabled = configurationSection.getBoolean("enabled");
+    this.players = configurationSection.getInt("players");
+  }
+
+  public String getName() {
+    return name;
+  }
+
+  public boolean isEnabled() {
+    return enabled;
+  }
+
+  public void setEnabled(boolean enabled) {
+    this.enabled = enabled;
+  }
+
+  public List<PGMMap> getMaps() {
+    return Collections.unmodifiableList(maps);
+  }
+
+  public int getPlayers() {
+    return players;
+  }
+
+  public void setPlayers(int players) {
+    this.players = players;
+  }
+
+  public void setPosition(int position) {
+    this.position = position;
+  }
+
+  public int getPosition() {
+    return position;
+  }
+
+  public void load() {
+    configurationSection
+        .getStringList("maps")
+        .forEach(
+            map -> {
+              final Optional<PGMMap> mapByNameOrId =
+                  PGM.get().getMapLibrary().getMapByNameOrId(map);
+              if (mapByNameOrId.isPresent()) maps.add(mapByNameOrId.get());
+              else {
+                PGM.get()
+                    .getLogger()
+                    .warning(
+                        "[Rotation] "
+                            + "["
+                            + name
+                            + "] "
+                            + map
+                            + " not found in map repo. Ignoring...");
+              }
+            });
+
+    Optional<PGMMap> nextMap =
+        PGM.get().getMapLibrary().getMapByNameOrId(configurationSection.getString("next_map"));
+    if (!nextMap.isPresent()) {
+      PGM.get()
+          .getLogger()
+          .log(
+              Level.SEVERE,
+              "Could not resolve next map from rotations. Resuming on initial position: 0");
+      this.position = 0;
+    } else {
+      this.position = getMapPosition(nextMap.get());
+    }
+  }
+
+  private int getMapPosition(PGMMap map) {
+    int count = 0;
+
+    for (PGMMap pgmMap : maps) {
+      if (pgmMap.getName().equals(map.getName())) break;
+      count++;
+    }
+
+    return count;
+  }
+
+  private void rotate() {
+    if (position + 1 == maps.size() || position + 1 > maps.size()) {
+      position = 0;
+    } else {
+      movePosition(1);
+    }
+  }
+
+  public void movePosition(int positions) {
+    if (((position + positions) + 1) > maps.size()) {
+      position = ((position + positions) + 1) % maps.size();
+    } else {
+      position = position + positions;
+    }
```

Comment:
Why isn't this just
`position = (position + positions) % maps.size();` ?
If you add an extra +1, you'll end at position 1 after finishing the rotation instead of position 0